src/services/textChanges.ts

```diff
@@ -283,7 +281,7 @@ namespace ts.textChanges {
         }
 
         // TODO (https://github.com/Microsoft/TypeScript/issues/21246): default should probably be useNonAdjustedPositions
-        public replaceRange(sourceFile: SourceFile, range: TextRange, newNode: Node, options: ChangeNodeOptions = {}) {
+        public replaceRange(sourceFile: SourceFile, range: TextRange, newNode: Node, options: InsertNodeOptions = {}) {

```

Comment:
As before, I personally find it strange to change parameter types of some, but not all, members of this family of functions.

Reply:
`interface ChangeNodeOptions extends ConfigurableStartEnd, InsertNodeOptions {}`. `ConfigurableStartEnd` determines how the start and end positions of the node parameter are adjusted. This function doesn't take a node as a parameter, so those options would be unused.